nixos/release.nix

```diff
@@ -121,7 +121,7 @@ let
     // {
       meta = {
         description = "NixOS system tarball for ${system} - ${stdenv.hostPlatform.linux-kernel.name}";
-        maintainers = map (x: lib.maintainers.${x}) maintainers;
```

Comment:
@wolfgangwalther we'll probably need to tighten the pattern to match on `meta = with lib; ` only. FYI

Another example of a problematic pattern would be:

```
{
pkgs,
... # no lib argument!
}:
... {
  meta = with pkgs.lib; {
  };
}
```

I've seen it a few times in the tree.

Reply:
I will spend some time on the refactoring rules this week. It's on my radar.